instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/TracingRequestWrapperBase.java

```diff
@@ -6,54 +6,44 @@
 package io.opentelemetry.instrumentation.awslambda.v1_0;
 
 import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import io.opentelemetry.sdk.OpenTelemetrySdk;
 import io.opentelemetry.sdk.autoconfigure.OpenTelemetrySdkAutoConfiguration;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.util.function.BiFunction;
 
 /**
  * Base abstract wrapper for {@link TracingRequestHandler}. Provides: - delegation to a lambda via
  * env property OTEL_INSTRUMENTATION_AWS_LAMBDA_HANDLER in package.ClassName::methodName format
  */
 abstract class TracingRequestWrapperBase<I, O> extends TracingRequestHandler<I, O> {
 
+  protected static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
   private final WrappedLambda wrappedLambda;
+  private final BiFunction<I, Class, Object> parameterMapper;
 
-  protected TracingRequestWrapperBase() {
-    this(OpenTelemetrySdkAutoConfiguration.initialize(), WrappedLambda.fromConfiguration());
+  protected TracingRequestWrapperBase(BiFunction<I, Class, Object> parameterMapper) {
```

Comment:
Nit: can be package private
```suggestion
  TracingRequestWrapperBase(BiFunction<I, Class, Object> parameterMapper) {
```

Reply:
I wanted to leave this package private for extension in downstream distros.